4.1/crud-operation-fetch.md

```diff
@@ -87,6 +87,28 @@ public function fetchUser() {
     }
 ```
 
+<a name="fetch-ajax-filter"></a>
+## Using FetchOperation with `select2_ajax` filter
+
+FetchOperation can also be used with `select2_ajax` filter. To make it work we need to setup aditional attributes in the filter. By default the `method` used by FetchOperation is `POST` while in filter is `GET`. We also need to tell backpack what attribute we want to show to the user. 
+
+```
+$this->crud->addFilter([
+  'name'        => 'category_id',
+  'type'        => 'select2_ajax',
+  'label'       => 'Category',
+  'placeholder' => 'Pick a category',
+  'method' => 'POST', //mandatory change
+  'select_attribute' => 'name' //the attribute that will be shown to the user by default 'name'
+  'select_key' => 'id' //by default is ID, change if your model use some other key
```

Comment:
```suggestion
  'method' => 'POST', // mandatory change
  // 'select_attribute' => 'name' // the attribute that will be shown to the user by default 'name'
  // 'select_key' => 'id' // by default is ID, change it if your model uses some other key
```

Should we let the default attributes commented? To encourage users to not use it if they are copy/pasting this code?

Reply:
💯 agree.